src/client/Microsoft.Identity.Client/Cache/Items/MsalAccessTokenCacheItem.cs

```diff
@@ -81,8 +85,13 @@ internal string TenantId
         internal string CachedAt { get; set; }
         internal string ExpiresOnUnixTimestamp { get; set; }
         internal string ExtendedExpiresOnUnixTimestamp { get; set; }
-        public string UserAssertionHash { get; set; }
+        internal string UserAssertionHash { get; set; }
  
+        /// <summary>
+        /// Used when the token is bound to a public / private key pair which is identified by a key id (kid). 
+        /// Currently used by PoP tokens
+        /// </summary>
+        internal string Kid { get; private set; }
```

Comment:
i know the spec calls it a "kid" (because they're trying to save json data transfer space).  Can we call it KeyId in our code?